images/logging-eventrouter.yml

```
@@ -1,15 +1,25 @@
+content:
+  source:
+    dockerfile: Dockerfile.rhel8
+    git:
+      branch:
+        # This repo does not follow the release- branch convention. Remove fallback when it does.
+        fallback: master
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/eventrouter.git
```

https://github.com/openshift-priv/eventrouter doesn't exist yet. We'll want DPTP to add this repo to its list of managed repos so that it gets synced to the private org.

In the meantime, we can point at the non-private org.
```suggestion
      # update to openshift-priv once that exists
      url: git@github.com:openshift/eventrouter.git
```